<h2>Description</h2>
<p>StV (HOR structural variant) annotation of alpha satellite higher-order repeats in T2T CHM13 human genome assembly release v1.1.</p>

<h2>Methods</h2>
<p>The StV track shows HOR structural variants (as opposed to sequence variants (SqVs). StVs can usually be shown as whole-monomer indels of the full-size (canonical) HOR. The track was generated for active HORs only according to the active array coordinates in the CenSat annotation, and was derived from the AS HOR annotation track. The main idea of generating the StV map was iterating through AS HOR annotation bed file and cutting after the last monomer in a HOR (the one with maximal monomer number in a given array) or before the first monomer (the one with number 1, e.g. S1C1/5/19H1L.1) if the last monomer was not present in a given StV. For S1C1/5/19H1L which forms the active HORs in centromeres 1, 5 and 19, the StVs with multiple 6/4-5 dimers within a HOR are shown as 1-5(_6/4_5){n}-6, where &#8216;n&#8217; is a number of the dimers in a HOR. All HORs are numbered (e.g. #1) starting from the left side of the array (the p-arm side) regardless of the strand orientation.</p>
<p>See details: Altemose et al., 2021, Supplementary Materials and Methods, Section 6: HOR unit and NRS prediction. HOR Structrual Variant (StV) prediction using CHM13 hor-monomer annotation.</p>
<p>See the complete <a href="https://docs.google.com/spreadsheets/d/1mO5-ZcaplNlH1go5mkplg_iGDVg6x6wDQ30AwDZBqd8" target="_blank">StV track statistics</a>.</p>

<!-- h2>Display Conventions and Configuration</h2 -->
<!-- if there are special conventions, like colors, describe them here -->
<!-- p></p -->

<h2>References</h2>
<p>Nicolas Altemose, Glennis Logsdon, Andrey V Bzikadze, Pragya Sidhwani, Sasha A Langley, Gina V. Caldas, Savannah J. Hoyt, Lev Uralsky, Fedor D. Ryabov, Colin Shew, Michael E.G. Sauria, Matthew Borchers, Ariel Gershman, Alla Mikheenko, Valery A. Shepelev, Tatiana Dvorkina, Olga Kunyavskaya, Mitchell R Vollger, Arang Rhie, Ann M. McCartney, Mobin Asri, Ryan Lorig-Roach, Kishwar Shafin, Sergey Aganezov, Daniel Olson, Leonardo Gomes de Lima, Tamara Potapova, Gabrielle A. Hartley, Marina Haukness, Peter Kerpedjiev, Fedor Gusev, Kristof Tigyi, Shelise Y. Brooks, Alice Young, Sergey Nurk, Sergey Koren, Sofie Salama, Benedict Paten, Evgeny I. Rogaev, Aaron M Streets, Gary H Karpen, Abby Dernburg, Beth A Sullivan, Aaron F Straight, Travis Wheeler, Jennifer L. Gerton, Evan Eichler, Adam M. Phillippy, Winston Timp, Megan Y. Dennis, Rachel J O'Neill, Justin M Zook, Michael Schatz, Pavel A Pevzner, Mark Diekhans, Charles H. Langley, Ivan A. Alexandrov, Karen H Miga. <i>bioRxiv</i> 2021.07.12.452052; doi: <a href="https://doi.org/10.1101/2021.07.12.452052" target="_blank">doi.org/10.1101/2021.07.12.452052</a></p>
<p>Uralsky, L. I.; Shepelev, V. A.; Alexandrov, A. A.; Yurov, Y. B.; Rogaev, E. I.; Alexandrov, I. A. Classification and Monomer-by-Monomer Annotation Dataset of Suprachromosomal Family 1 Alpha Satellite Higher-Order Repeats in Hg38 Human Genome Assembly. <i>Data Brief</i> 2019, 24, 103708. doi: <a href="https://doi.org/10.1016/j.dib.2019.103708" target="_blank">doi.org/10.1016/j.dib.2019.103708</a>.</p>

<h2>Release history</h2>
<ol>
  <li>StV10 - First public version of the StV annotation.
  <li>2021-10-22 t2t-CHM13.release_v1.1 - Annotation of CHM13 v1.1 assembly.
</ol>

<h2>Contact</h2>
<p>Contact <a href="mailto:fedorrik1@gmail.com">Fedor Ryabov &lt;fedorrik1@gmail.com&gt;</a></p>
<p>Contact <a href="mailto:uralsky@rogaevlab.ru">Lev I. Uralsky &lt;uralsky@rogaevlab.ru&gt;</a></p>
<p>Contact <a href="mailto:ivanalx@hotmail.com">Ivan A. Alexandrov &lt;ivanalx@hotmail.com&gt;</a></p>

<h2>Credits</h2>
<p>Fedor Ryabov (Moscow Polytechnic University, Russia)</p>
<p>Lev I. Uralsky (1. Sirius University of Science and Technology, Sochi, Russia; 2. Vavilov Institute of General Genetics, Moscow, Russia)</p>
<p>Ivan A. Alexandrov (1. Vavilov Institute of General Genetics, Moscow, Russia; 2. Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia)</p>
